cuebot/src/test/java/com/imageworks/spcue/test/dao/postgres/DispatcherDaoFifoTests.java

```diff
@@ -212,6 +211,7 @@ public void testPortionSorted() throws Exception {
     @Test
     @Transactional
     @Rollback(true)
+    @Ignore
```

Comment:
I don't really understand this unit test. I dug into it a bit and I don't see an obvious problem, but the `assertNotEquals(jobs, sortedJobs)` check fails for me sometimes in an ARM environment. After looking at the dispatch SQL query, I don't see why `jobs` and `sortedJobs` should be different. The jobs are all launched with the same priority, so it's expected to get them back in the same order, right?

@DiegoTavares Any idea here?

Reply:
This test doesn't make much sense to me. I guess the first assert with sortedJobs can be removed as it's not guaranteed both lists will be different.